modules/op-creating-pipeline-tasks.adoc

```diff
@@ -0,0 +1,62 @@
+// Ths module is included in the following assembly:
+//
+// *openshift_pipelines/op-creating-applications-with-cicd-pipelines.adoc
+
+[id="creating-pipeline-tasks_{context}"]
+= Creating Pipeline Tasks
+
+[discrete]
+.Procedure
+
+. Install the `apply-manifests` and `update-deployment` Tasks from the `pipelines-tutorial` repository, which contains a list of reusable Tasks for Pipelines:
++
+----
+$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/release-tech-preview-1/01_pipeline/01_apply_manifest_task.yaml
+$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/release-tech-preview-1/01_pipeline/02_update_deployment_task.yaml
+----
+
+. Verify the Tasks you created:
++
+----
+$ tkn task list
+
+NAME                DESCRIPTION   AGE
```

Comment:
We recently updated our guidance:

- Do not use more than one command per code block.

- Separate a command and its related output into individual code blocks.

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks

Please separate command and output.